server/src/com/google/refine/Refine.java

```diff
@@ -516,11 +516,11 @@ private void openBrowserFallback() throws IOException {
         Runtime rt = Runtime.getRuntime();
 
         if (SystemUtils.IS_OS_WINDOWS) {
-            rt.exec("rundll32 url.dll,FileProtocolHandler " + uri);
+            rt.exec(new String[] { "rundll32 url.dll,FileProtocolHandler ", String.valueOf(uri) });
```

Comment:
Looking at the documentation of this method, it is not clear to me if the array of strings is expected to match the canonical tokenization of the command, or if the strings are just internally concatenated together anyway.
So for this case I wonder if it should not rather be 
```suggestion
            rt.exec(new String[] { "rundll32 ", "url.dll,FileProtocolHandler ", String.valueOf(uri) });
```
And I wonder if we can remove the trailing spaces around the strings (also below).

Did someone test if this still works on Windows? @thadguidry perhaps?

Reply:
@wetneb I tried running tests after checking out PR and seems I get failures.  Here's the surefire test output:
[tests.zip](https://github.com/OpenRefine/OpenRefine/files/10053209/tests.zip)

Reply:
Looks like it's something to do with the space in my repo path (which is perfectly normal to have).  
`E:\GitHub Repos\OpenRefine`
but after running the tests, I notice that this new folder with empty subfolders structure was made??? weird.
`E:\GitHub%20Repos\OpenRefine\main\target\test-classes`

Reply:
It is curious that this PR would change anything to that, I do not think that the lines affected by the PR are coverered by tests. Do the tests pass on the master branch? If not I would open a specific issue about this.
Did you try running OpenRefine interactively on this branch? I would be interested to know if the browser opens automatically.

Reply:
OS detected and browser (my default Edge on Windows 11) opened just fine.  Anything else to test on this branch?
Yeah, I'll open an issue for failing tests on Windows.

Reply:
Thanks, that's all I wanted to know :)